extra_data/reader.py

```diff
@@ -117,6 +118,7 @@ def __init__(
             }
         self._sources_data = sources_data
 
+        self._alias_files = [] if alias_files is None else alias_files
```

Comment:
It makes sense in the context of this PR, but could you add a comment this is only for tracking purposes? The actual aliases are still carried via the `aliases` dict after all.